openmetadata-ui/src/main/resources/ui/playwright.config.ts

```diff
@@ -52,9 +52,9 @@ export default defineConfig({
     baseURL: process.env.PLAYWRIGHT_TEST_BASE_URL || 'http://localhost:8585',
 
     /* Collect trace when retrying the failed test. See https://playwright.dev/docs/trace-viewer */
-    trace: 'on-first-retry',
+    trace: 'retry-with-trace',
     /* Screenshot on failure. */
-    screenshot: 'only-on-failure',
+    screenshot: 'off',
   },
```

Comment:
I recall that we chose not to modify this setting. Could you clarify the reason for this change?

Reply:
Reverted